src/css/components/_progress.scss

```diff
@@ -27,6 +27,12 @@
 /* We need an increased hit area on hover */
 .video-js .vjs-progress-control:hover .vjs-progress-holder { font-size: 1.666666666666666666em }
 
+/* Also show the current time tooltip */
+.video-js .vjs-progress-control:hover .vjs-play-progress:after {
+  display: block;
+  font-size: 0.6em;
```

Comment:
If someone wants it always up (a la Vimeo), they could just set the main
one to display: block. If we don't change the font size on hover, it's
comically huge.
On Wed, Jun 10, 2015 at 6:33 PM Gary Katsevman <notifications@github.com>
wrote:

> In src/css/components/_progress.scss
> <https://github.com/videojs/video.js/pull/2255#discussion_r32183524>:
>
> > @@ -27,6 +27,12 @@
> >  /* We need an increased hit area on hover */
> >  .video-js .vjs-progress-control:hover .vjs-progress-holder { font-size: 1.666666666666666666em }
> >
> > +/* Also show the current time tooltip */
> > +.video-js .vjs-progress-control:hover .vjs-play-progress:after {
> > +  display: block;
> > +  font-size: 0.6em;
>
> why does the font-size change?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/videojs/video.js/pull/2255/files#r32183524>.
>